lib/exception_hunter.rb

```diff
@@ -1,4 +1,5 @@
 require 'exception_hunter/engine'
+require 'exception_hunter/railtie' if defined?(Rails)
```

Comment:
Rails should always be defined
```suggestion
require 'exception_hunter/railtie'
```